pkg/cmd/cli/cmd/logs.go

```diff
@@ -163,6 +168,13 @@ func (o *OpenShiftLogsOptions) RunLog() error {
 		}
 		return o.runLogsForBuild(build)
 
+	case "dc", "deploymentconfig", "deploymentconfigs":
```

Comment:
> We have that in the RESTMapper already - why do we need a second location.

See https://github.com/openshift/origin/pull/4947#discussion_r41714595

> does VersionAndKindForResource handle singlar resources

Yes, it does.